### ISSUE TITLE
Cannot disable menu item

### ISSUE THREAD

**Issue by alevlako** [bosskmk/pluto_menu_bar#9]
I didn't notice any difference when "enable" true or false. In both cases, the appearance of the menu is the same and it works. I expected to see the disabled menu item when enable is false.

```
PlutoMenuItem(title: 'Help', children: [
        PlutoMenuItem(
          title:  'Tutorial',
          onTap: () => message(context, 'Tutorial'),
          **enable**: true,
        ),        
      ])
```

**Comment by bosskmk** [comment]
Valid only when `PlutoMenuBar.mode` is `PlutoMenuBarMode.tap`.
This is a function prior to the addition of `PlutoMenuBarMode.hover` mode and should be applied in hover mode as well.
We will add it in the next version.
thank you

**Comment by alevlako** [comment]
You are right, I am using the hover mode, as more native for windows desktop in my opinion.
I am very glad to found your plagin, it looks more pretty, then many others. I will look forward to adding the discussed functionality to the hover mode. Thank you!
